feature/login/src/main/java/com/iteneum/login/presentation/LoginView.kt

```diff
@@ -45,15 +42,18 @@ import com.iteneum.login.R
  * @author Jesus Lopez
  */
 @Composable
-fun LoginView() {
+fun LoginView(loginViewModel: LoginViewModel = hiltViewModel()) {
     val sizes = LeasePertTheme.sizes
 
     ConstraintLayout(
         modifier = Modifier
             .fillMaxSize()
 
     ) {
-        val (logo, mail, password, tonalButton, loginWith, socialNetworkLogins, registerButton) = createRefs()
+        val (logo, emailField, passwordField, loginButton, loginWith, socialNetworkLogins, registerButton) = createRefs()
```

Comment:
You should use remember variables to set it to the viewModel

Reply:
Those variables are needed to create the references for the constraintLayout for the composable view, should I still proceed to change them to remember variables?